oct/tests/api/registration_pattern.py

```diff
@@ -0,0 +1,25 @@
+from typing import List
+import requests
+
+
+class RegistrationAPI:
+    def __init__(self, full_name: List[str], email: str, telephone: str, password: str):
+        self._full_name = full_name
+        self._email = email
+        self._telephone = telephone
+        self._password = password
+
+    def account_registration(self) -> None:
```

Comment:
@AlekseyDidilov, please update method to the following
```suggestion
    def registration_response(self) -> str:
```